### PR TITLE
Fixes #17250: Webapp does not regerate policies when webdav password is changed, breaking inventories after 5.0 -> 6.0 upgrade

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/17250

Replacing previous PR: https://github.com/Normation/rudder/pull/3217

### CODE REVIEW

**Comment by ncharles** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala [hunk 1]
```diff
@@ -174,57 +173,16 @@ final class AsyncDeploymentActor(
     )
   }
 
-  // That flag file is used to determine if a policy update was running when the webapp stopped so we can launch a new one when starting
-  val policyUpdateRunningFlagPath = "/opt/rudder/etc/policy-update-running"
```
Comment (fanf) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala:178]: Shouldn't we delete that one during `rudder-upgrade`?

**Review by fanf — CHANGES_REQUESTED** [review]
Appart from two small remark, seems good! Is there only rudder that set file `/opt/rudder/etc/policy-update-running`?

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala (hunk 1 above)
Comment (ncharles) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala:178]: good point

**Comment by ncharles** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala (hunk 1 above)
Comment (ncharles) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala:178]: done

**Review by fanf — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**